### DISCUSSION TITLE
独立开发者出海周刊（第30期） | 信息差——独立开发者出海周刊

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/gapis.money#74] [General]
# 独立开发者出海周刊（第30期） | 信息差——独立开发者出海周刊

Knowledge is power, info-gap is money!「信息差——独立开发者出海周刊」是一个帮助独立开发者缩小信息差的技术周刊。

https://gapis.money/weekly/2024-12-16_030

**Comment by JiuRanYa** [comment]
我是IconEase的作者，非常感谢大佬能够推荐～

**Comment by WymanY** [comment]
哈哈哈，只要这个周刊还在更新，只要看过，我就给一个大大的 👍

**Comment by lanxkk** [comment]
很棒的一个周刊！！

**Comment by wolfsmith** [comment]
Markdown 写起来舒服，但最后要交 Word 经常折腾。我发现一个在线小工具可以直接把 Markdown 转成 Word，很省事（https://www.markdown-to-word.online/）。对于写文档的人挺好用的，大家可以根据需要试一下。
